css/border-width/demos/basic/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+  <title>Блок текста — border-width — Дока</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="preconnect" href="https://fonts.gstatic.com/" crossorigin>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto&display=swap">
+  <style>
+    * {
+      padding: 0;
+      margin: 0;
+      font-family: "Roboto", sans-serif;
+      box-sizing: border-box;
+    }
+
+    html {
+      height: 100%;
+    }
+
+    body {
+      height: 100%;
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      background-color: #dfdfdf;
+    }
+
+    .text {
+      font-size: 16px;
+      padding: 5px 20px;
+      border-style: solid;
+      border-width: 3px;
+    }
+  </style>
+</head>
+<body>
+  <p class="text">Блок текста, обведённый рамкой</p>
```

Comment:
```suggestion
   p {
      font-size: 16px;
      padding: 5px 20px;
      border-style: solid;
      border-width: 3px;
    }
  </style>
</head>
<body>
  <p>Блок текста, обведённый рамкой</p>
```

С учётом, что в демке всего один элемент задавать ему класс кажется избыточным.